test/specs/collections/Breadcrumb/BreadcrumbSection-test.js

```diff
@@ -0,0 +1,84 @@
+import React from 'react'
+import Breadcrumb from 'src/collections/Breadcrumb/Breadcrumb'
+import * as common from 'test/specs/commonTests'
+import sandbox from 'test/utils/Sandbox-util'
+
+describe('BreadcrumbSection', () => {
+  common.isConformant(Breadcrumb.Section)
+  common.rendersChildren(Breadcrumb.Section)
+
+  describe('active', () => {
```

Comment:
For props that convert to classNames, we have `propUtils` for extracting the classes and `commonTests` for testing them.  Have a look at CONTRIBUTING.md under the [Prop Utils](https://github.com/TechnologyAdvice/stardust/blob/master/CONTRIBUTING.md#prop-utils) and the following section [Testing className](https://github.com/TechnologyAdvice/stardust/blob/master/CONTRIBUTING.md#testing-classname) for how to use them.  Let me know if you have any questions after that.